README.md

```diff
@@ -46,7 +46,9 @@ If your app depends on the old unmigrated atmosphere packages than you also need
 
 ## The simplest possible test
 
-Basically, you run the tests with `gagarin` command within you project root.
+First run your meteor app with command `meteor`. 
```

Comment:
@parhelium In fact this is not required ... if meteor is not running in the background, gagarin will build your project.